### PR TITLE
Circle Payment

### PR DESCRIPTION
**Pull request by roienatan**
resolves: #750

### CODE REVIEW

File: packages/web/src/shared/constants/shared.tsx [hunk 1]
```diff
@@ -13,6 +13,8 @@ export const COMMON_APP_GOOGLE_PLAY_LINK = "https://play.google.com/store/apps/d
 
 export const CONTACT_EMAIL = "hi@common.io";
 
+export const CIRCLE_PAY_URL = "ttps://api-sandbox.circle.com/v1";
```
Comment (watchtower314) [inline packages/web/src/shared/constants/shared.tsx]: @roienatan did you add the `h` locally? 😅

Comment (roienatan) [reply packages/web/src/shared/constants/shared.tsx]: Yes haha
Will update here soon also.

File: packages/web/src/containers/Common/components/CommonDetailContainer/MembershipRequestModal/MembershipRequestPayment.tsx [hunk 2]
```diff
@@ -1,23 +1,102 @@
...
+
 export default function MembershipRequestPayment(props: IStageProps) {
   const { userData, setUserData } = props;
-  const [card, setCard] = useState(0);
-  const [expiration, setExpiration] = useState("");
+  const [card_number, setCardNumber] = useState(0);
+  const [expiration_date, setExpirationDate] = useState("");
   const [cvv, setCvv] = useState(0);
 
+  const createJoinProposal = async (formData: any) => {
```
Comment (razoorka) [inline packages/web/src/containers/Common/components/CommonDetailContainer/MembershipRequestModal/MembershipRequestPayment.tsx:34]: Can be moved outside from MembershipRequestPayment

Comment (roienatan) [reply packages/web/src/containers/Common/components/CommonDetailContainer/MembershipRequestModal/MembershipRequestPayment.tsx:34]: Thanks. Anyway it's still draft and I'm trying to make the basic functionality working first. After that will organize the code in a better way.

**Pull request merged**